shell/platform/windows/flutter_window_win32_unittests.cc

```diff
@@ -246,21 +242,24 @@ class TestFlutterWindowsView : public FlutterWindowsView {
   }
 
  protected:
-  void RegisterKeyboardHandlers(
+  std::unique_ptr<KeyboardHandlerBase> CreateKeyboardKeyHandler(
       flutter::BinaryMessenger* messenger,
       flutter::KeyboardKeyHandler::EventDispatcher dispatch_event,
       flutter::KeyboardKeyEmbedderHandler::GetKeyStateHandler get_key_state)
       override {
-    auto spy_key_event_handler = std::make_unique<SpyKeyboardKeyHandler>(
+    auto spy_key_event_handler = new SpyKeyboardKeyHandler(
```

Comment:
Why not keep the `make_unique` here? That way if someone later adds code that throws an exception or returns before the end of the function, this won't leak.